cmd/convertor/builder/overlaybd_builder.go

```diff
@@ -0,0 +1,192 @@
+/*
+   Copyright The Accelerated Container Image Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package builder
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+
+	"github.com/containerd/accelerated-container-image/pkg/snapshot"
+	"github.com/opencontainers/go-digest"
+	specs "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+)
+
+const (
+	// labelKeyOverlayBDBlobDigest is the annotation key in the manifest to
+	// describe the digest of blob in OverlayBD format.
+	//
+	// NOTE: The annotation is part of image layer blob's descriptor.
+	labelKeyOverlayBDBlobDigest = "containerd.io/snapshot/overlaybd/blob-digest"
```

Comment:
This specific annotation is also present in pkg/snapshot/overlay.go  and pkg/convertor.go, I assume it's unlikely to change in the future, but it might make sense to either put the annotations in a common file or making the constants public to avoid redefinition in the future and possible inconsistencies if there is a change at some point.

Reply:
I think we should manage all annotations to a standalone file like 'accelearated-container-image/pkg/snapshot/labels.go' ?

Reply:
I have moved all annotations to `accelearated-container-image/pkg/label/label.go`